source/code-snippets/collation/collection-collation.js

```diff
@@ -0,0 +1,18 @@
+// ignored first line
+
+// start create collection with collation
+// Create the collection with a collation
+db.createCollection("souvenirs", {
+  collation: { locale: "fr_CA" },
+});
+// end create collection with collation
+
+// start collection query without collation
+collection.find({type: "photograph"});
```

Comment:
I think using the collection name itself (for instance, `souvenirs.find` here, `names.find` for the last example) might make it a little easier for readers to follow since they don't have all the extra context now of the db/collection object instantiation now. Not sure if it's necessary, though.

Reply:
One thing we may need to unify is the usage example code snippet and CRUD guide conventions for variable naming of the db and connection. In all the usage examples we use the following convention:
 ``` 
const db = client.db('sample_mflix');
const collection = db.collection('movies');
```
In your CRUD guide examples, I see a mix:
```
const database = client.db("sample_mflix");
const collection = database.collection("movies");
```
and
```
const database = client.db("test");
const orders = database.collection("orders");
```

I don't think there's a good answer, but we should be consistent. I'll leave this as-is for now and make a mental note for later.

Reply:
Really good point -- we'll table the discussion for later. I'll create a ticket for this -- is there a "Node Enhancements" epic I can throw it in?

Reply:
https://jira.mongodb.org/browse/DOCSP-8820 @nathan-contino-mongo